addons/mrp/models/mrp_production.py

```diff
@@ -65,7 +65,9 @@ def _get_default_date_planned_finished(self):
     def _get_default_date_planned_start(self):
         if self.env.context.get('default_date_deadline'):
             return fields.Datetime.to_datetime(self.env.context.get('default_date_deadline'))
-        return datetime.datetime.now()
+        regex_to_round_up = re.compile("[0-9]+-[0-9]+-[0-9]+T[0-9]+")
+        datetime_rounded_by_hour = regex_to_round_up.match(datetime.datetime.now().isoformat()).group()
+        return datetime.datetime.strptime(datetime_rounded_by_hour, "%Y-%m-%dT%H") + datetime.timedelta(hours=1)
```

Comment:
This test doesn't pass: https://runbot.odoo.com/runbot/build/8801609 because I have changed the expected behavior of how **workorder.date_planned_start** are calculate.
And I'm wondering if I need to change **workorder.date_planned_finished** too. I guess so.

Reply:
Exactly